src/subtyping.md

```diff
@@ -15,5 +15,75 @@ fn bar<'a>() {
     let t: &'a str = s;
 }
 ```
-Since `'static` "lives longer" than `'a`, `&'static str` is a subtype of
-`&'a str`.
+
+Since `'static` outlives the lifetime parameter `'a`, `&'static str` is a
+subtype of `&'a str`.
+
+[Higher-ranked]&#32;[function pointers] and [trait objects] have another
```

Comment:
why is this escaped space necessary? otherwise markdown parses it as a reference?

Reply:
Markdown would read it as a link with text of "[Higher ranked]" and a link reference to "function pointers" instead of as two separate links.